src/ghc.jl

```diff
@@ -0,0 +1,126 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    GHC(kernel,merge_crit;k=35,vars=nothing)
+
+A method for partitioning spatial data into clusters 
+using Geostatistical Hierarchical Clustering (GHC).
+
+## Parameters
+
+* `kernel`      - type of kernel used for estimator
+* `merge_crit`  - strategy to specify difference between clusters
+* `k`           - minimum number of observations contained in support 
+                    of kernel function centered at each point. 
+                    Defaults to 35.
+* `vars`        - Variables (or features) to consider (default to all)
+
+## References
+* F. Fouedjio. 2016. [A hierarchical clustering method for multivariate geostatistical data]
+  (https://www.sciencedirect.com/science/article/abs/pii/S2211675316300367)
+"""
+
+uniform(z;λ) = (z ≤ λ)
+triangular(z;λ) = (λ - z)*(z ≤ λ)
+Epanechnikov(z;λ) = (λ^2 - z^2)*(z ≤ λ )
+Gaussian(z;λ) = exp((-z^2)/(2λ^2))
```

Comment:
The docstring should come right before the GHC struct otherwise Julia won´t assign it correctly.